persistent-sqlite/Database/Persist/Sqlite.hs

```diff
@@ -470,6 +470,8 @@ migrate' allDefs getter val = do
   where
     def = val
     table = getEntityDBName def
+    schema = getEntitySchema def
+    sqliteMaster = maybe "sqlite_master" (\schema' -> escapeS schema' <> ".sqlite_master") schema
```

Comment:
There is one copy of the `sqlite_master` table in each schema, and it only contains information about the tables in its own schema.